### ISSUE TITLE
awesome-gemini in geminispace?

### ISSUE THREAD

**Issue by guzzisti** [kr1sp1n/awesome-gemini#20]
I found a mirror of this site in geminispace at gemini://gemini.susa.net/gen/awesome-gemini_kr1sp1n.gmi, but its outdated.

Is there an official mirror in geminispace somewhere? If not, we should definitely do so.

**Comment by kr1sp1n** [comment]
@guzzisti Good idea! I will try to transform the `README.md` to Gemini and then publish it on https://kr1sp1n.flounder.online/awesome.gmi as often as possible. Maybe I will automate this via GitHub actions.

**Comment by guzzisti** [comment]
Unfortunately the linked site is empty (on both https and gemini).

**Comment by kr1sp1n** [comment]
I did not find the time to transform the README.md in this repo to a gemini page and to upload it to flounder.online. When I finally did this I will inform you and close this issue.

**Comment by kr1sp1n** [comment]
Has anyone a tip or a tool to transform Markdown to gemtext? Thanks in advance.

**Comment by bacardi55** [comment]
From this list, I've used this one with success to transform my blog post in .md to .gmi files
https://github.com/makeworld-the-better-one/md2gemini

**Comment by DhruvaSambrani** [comment]
Hmmm, if only there were a list which had some tools for gemini conversions.....

**Comment by playmobilmeister** [comment]
> [@Guzzisti](https://github.com/Guzzisti) Good idea! I will try to transform the `README.md` to Gemini and then publish it on https://kr1sp1n.flounder.online/awesome.gmi as often as possible. Maybe I will automate this via GitHub actions.

Hi @kr1sp1n , and thank you for maintaining the list. Thanks to you, I found some useful software. It looks like the gemini version is not regularly updated. Do you know of any mirrors in Geminispace that update regularly?

**Comment by bcasiello** [comment]
I maintain a Geminispace mirror at gemini://gem.casiello.net/awesome-gemini.gmi

It is currently up to date (barring anything I messed up when transcribing), and my intention is to keep it that way as much as possible.
